### PR TITLE
DAC6-341 - Check your answers page for individuals.

### PR DESCRIPTION
**Pull request by iservport**

### CODE REVIEW

File: app/utils/rows/RowBuilder.scala [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    val message = MessageInterpolators(StringContext.apply(s"$msgKey.checkYourAnswersLabel")).msg()
+    Row(
+      key     = Key(message, classes = Seq("govuk-!-width-one-half")),
+      value   = Value(content),
+      actions = List(
+        Action(
+          content            = msg"site.edit",
+          href               = href,
+          visuallyHiddenText = Some(msg"site.edit.hidden".withArgs(message))
+        )
```
Comment (helensoniega) [inline app/utils/rows/RowBuilder.scala:49]: Add ids to change links by including 'attributes = Map("id" -> "someID")' to the Action

File: test/controllers/IndividualAddressControllerSpec.scala [hunk 2]
```diff
@@ -218,5 +219,35 @@ class IndividualAddressControllerSpec extends SpecBase with MockitoSugar with Nu
...
+          .overrides(
+            bind[Navigator].toInstance(new FakeNavigator(routes.IndividualCheckYourAnswersController.onPageLoad(), mode = CheckMode)),
+            bind[SessionRepository].toInstance(mockSessionRepository)
+          )
+          .build()
+
+      val request =
+        FakeRequest(POST, individualDateOfBirthRouteCheck.url)
+          .withFormUrlEncodedBody(validData.toList:_*)
+
```
Comment (helensoniega) [inline test/controllers/IndividualAddressControllerSpec.scala:242]: Maybe rename the unit test? Answer isn't changed but the mode is changed to CheckMode. You can also move this test to NavigatorSpec?

File: test/controllers/IndividualDateOfBirthControllerSpec.scala [hunk 3]
```diff
@@ -210,5 +211,35 @@ class IndividualDateOfBirthControllerSpec extends SpecBase with MockitoSugar wit
 
       application.stop()
     }
+
+    "must redirect to the Check your answers page when user change their answer" in {
```
Comment (helensoniega) [inline test/controllers/IndividualDateOfBirthControllerSpec.scala:215]: Same here

File: test/controllers/IndividualNameControllerSpec.scala [hunk 4]
```diff
@@ -167,5 +168,33 @@ class IndividualNameControllerSpec extends SpecBase with MockitoSugar with Nunju
 
       application.stop()
     }
+
+    "must redirect to the Check your answers page when user change their answer" in {
```
Comment (helensoniega) [inline test/controllers/IndividualNameControllerSpec.scala:172]: Same here

File: test/controllers/IndividualPlaceOfBirthControllerSpec.scala [hunk 5]
```diff
@@ -190,5 +192,35 @@ class IndividualPlaceOfBirthControllerSpec extends SpecBase with MockitoSugar wi
 
       application.stop()
     }
+
+    "must redirect to the Check your answers page when user change their answer" in {
```
Comment (helensoniega) [inline test/controllers/IndividualPlaceOfBirthControllerSpec.scala:196]: Same here

File: test/controllers/IsIndividualAddressKnownControllerSpec.scala [hunk 6]
```diff
@@ -194,5 +194,35 @@ class IsIndividualAddressKnownControllerSpec extends SpecBase with MockitoSugar
 
       application.stop()
     }
+
+    "must redirect to the Check your answers page when user change their answer to `No`" in {
```
Comment (helensoniega) [inline test/controllers/IsIndividualAddressKnownControllerSpec.scala:198]: Same here

File: test/controllers/IsIndividualPlaceOfBirthKnownControllerSpec.scala [hunk 7]
```diff
@@ -194,5 +194,35 @@ class IsIndividualPlaceOfBirthKnownControllerSpec extends SpecBase with MockitoS
 
       application.stop()
     }
+
+    "must redirect to the Check your answers page when user change their answer to `No`" in {
```
Comment (helensoniega) [inline test/controllers/IsIndividualPlaceOfBirthKnownControllerSpec.scala:198]: Last one

File: app/utils/rows/RowBuilder.scala (hunk 1 above)
Comment (iservport) [reply app/utils/rows/RowBuilder.scala]: Added.

File: test/controllers/IndividualAddressControllerSpec.scala (hunk 2 above)
Comment (iservport) [reply test/controllers/IndividualAddressControllerSpec.scala]: Moved tests to CheckModeNavigatorSpec.

**Review by helensoniega — APPROVED** [review]

**Comment by iservport** [comment]
Moved to a new PR because of a bug.

**Pull request closed without merging**